src/estimation/fdr/bh.rs

```diff
@@ -37,12 +37,12 @@ pub fn control_fdr<E: Event, W: io::Write>(
     calls: &mut bcf::Reader,
     null_calls: &mut bcf::Reader,
     writer: &mut W,
-    event: &E,
+    events: &Vec<E>,
```

Comment:
This should be simply a slice: `&[E]`. A vector will be coerced into that automatically.